BlocksForAll/Assets.xcassets/Orange.imageset/Contents.json

```diff
@@ -2,20 +2,14 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "Orange.pdf",
-      "scale" : "1x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "2x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "3x"
+      "filename" : "Orange.pdf"
```

Comment:
should be:
5-14
      "filename" : "Orange.pdf"
      "scale" : "1x"
    },
    {
      "idiom" : "universal",
      "scale" : "2x"
    },
    {
      "idiom" : "universal",
      "scale" : "3x"